aiohttp/helpers.py

```diff
@@ -1065,14 +1065,37 @@ def parse_http_date(date_str: Optional[str]) -> Optional[datetime.datetime]:
     return None
 
 
+def must_be_empty_body(method: str, code: int) -> bool:
+    """Check if a request must return an empty body."""
+    return (
+        status_code_must_be_empty_body(code)
+        or method_must_be_empty_body(method)
+        or (code == 200 and method.upper() == hdrs.METH_CONNECT)
```

Comment:
I believe it was anything in the 2xx range, not just 200 itself.

Reply:
Will hunt for it in the RFC and adjust as needed as well as the other PR

Reply:
> A server MUST NOT send a Content-Length header field in any response with a status code of [1xx (Informational)](https://www.rfc-editor.org/rfc/rfc9110.html#status.1xx) or [204 (No Content)](https://www.rfc-editor.org/rfc/rfc9110.html#status.204). A server MUST NOT send a Content-Length header field in any [2xx (Successful)](https://www.rfc-editor.org/rfc/rfc9110.html#status.2xx) response to a CONNECT request ([Section 9.3.6](https://www.rfc-editor.org/rfc/rfc9110.html#CONNECT)).

Yes. 2xx 👍

Reply:
adjusted in 286a1084fd75344bda821116d0109e152dcaf6d5